image/templates/helm/shared/config-templates/scanner/config.yaml.tpl

```diff
@@ -9,6 +9,7 @@
 
 scanner:
   centralEndpoint: https://central.{{ .Release.Namespace }}.svc
+  sensorEndpoint: sensor.{{ .Release.Namespace }}.svc
```

Comment:
I think the `centralEndpoint` schema should be removed instead, because it's treated by `urlfmt.FormatURL` [in Scanner](https://github.com/stackrox/scanner/blob/45d24fa0b2f69e195375ed0c5d2843238625bab9/pkg/updater/genesis_manifests.go#L63).

> We will need to ensure people know this field is added to Scanner's config map, what it means, and any other relevant data: for example, at the moment, it seems like the scheme should not be specified.

Glad to it, let me know where configmap doc lives, and what is the right audience (I can guess all customers would be interested by I need confirmation).

Reply:
The reason I recommended adding the scheme to `sensorEndpoint` is because we would need to document a migration path for customers if we modify a pre-existing value. Actually, I wonder if we would have to document a migration path if we add a new value like this as well. I'm not 100% sure. I'd ask around for someone who knows how our upgrade process works well. We can sync up on Slack about this